python/cudf/cudf/core/dataframe.py

```diff
@@ -4802,6 +4815,25 @@ def to_pandas(self, nullable=False, **kwargs):
         2  2  0
         >>> type(pdf)
         <class 'pandas.core.frame.DataFrame'>
+
+        Use ``nullable`` parameter to get Pandas Nullable dtypes:
+
+        >>> df = cudf.DataFrame({'a': [0, None, 2], 'b': [True, False, None]})
+        >>> df
+              a      b
+        0     0   True
+        1  <NA>  False
+        2     2   <NA>
+        >>> pdf = df.to_pandas(nullable=True)
+        >>> pdf
+              a      b
+        0     0   True
+        1  <NA>  False
+        2     2   <NA>
+        >>> pdf.dtypes
+        a      Int64
+        b    boolean
+        dtype: object
```

Comment:
Addressing the docstring, I think it's also interesting to showcase the situation where `nullable=False`

Reply:
Done